pkgs/by-name/af/affine/package.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, writeText
+, fetchurl
+, stdenvNoCC
+, copyDesktopItems
+, makeDesktopItem
+, unzip
+, bash
+, electron
+, commandLineArgs ? ""
```

Comment:
Why do we need to rebuild the entire package just to change the arguments to the wrapper?

Reply:
Electron applications usually need this. It allows user to pass extra arguments to electron. For example, `--enable-wayland-ime` is needed if someone wants to use input method in wayland.